pom.xml

```diff
@@ -452,6 +460,7 @@
                      javax.sql.rowset.spi,
                      com.codahale.metrics;resolution:=optional,
                      com.codahale.metrics.health;resolution:=optional,
+                     io.dropwizard.metrics5;resolution:=optional,
```

Comment:
I wasn't quite sure what this was, but given the pre-existing `codahale` entries, I figured this was appropriate to add to help with your release process.